src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/AdjournHearingsWindowMappingTest.java

```diff
@@ -0,0 +1,187 @@
+package uk.gov.hmcts.reform.sscs.helper.mapping;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.TestInstance.Lifecycle.PER_CLASS;
+
+@TestInstance(PER_CLASS)
```

Comment:
Is there a reason why you picked per class?

Reply:
Only because it is the one that makes the tests run and pass